extensions/libfilesystem/libfilesystem.gml

```diff
@@ -1,57 +1,57 @@
-#define get_temp_directory
```

Comment:
I think because some of this was edited on mac/linux IDE, they treat new lines differently so it must've removed the use of carriage return and replaced it with just line feed by itself.